themes/bootstrap3/templates/search/searchbox.phtml

```diff
@@ -163,3 +166,29 @@
   </form>
 <?php endif; ?>
 
+<?php
+$js = <<<JS
+        $(document).ready(function searchFormResetHandler() {
```

Comment:
Would this make more sense in common.js rather than coded into the template? If you use the .js file you can better take advantage of caching.